deepchem/models/torch_models/grover.py

```diff
@@ -1,9 +1,119 @@
 import torch
 import torch.nn as nn
+import numpy as np
 from typing import List
 from deepchem.models.torch_models.grover_layers import GroverTransEncoder
 
 
+class GroverFinetune(nn.Module):
```

Comment:
Can you add this class to `layers.rst`?

Reply:
Added to docs, I also added the test which I forgot to commit in 4633b41